analysis/metrics.py

```diff
@@ -192,32 +199,53 @@ class ValueErrors:
 
     Attributes:
         bounding_errors: contribution bounding errors.
-        bias: averaged across partitions E(dp_value - actual_value).
+        mean: averaged across partitions E(dp_value - actual_value), aka
+         statistical bias.
         variance: averaged across partitions Var(dp_value - actual_value).
         rmse: averaged across partitions sqrt(E(dp_value - actual_value)^2).
         l1: averaged across partitions E|dp_value - actual_value|.
         with_dropped_partitions: error which takes into consideration partitions
           dropped due to partition selection. See example below.
     """
     bounding_errors: ContributionBoundingErrors
-    bias: float
+    mean: float
     variance: float
+
     rmse: float
     l1: float
 
-    # The following error metrics include error from dropped partitions.
-    #
-    # Consider the following example with a single partition to see how they are
-    # different from abs/rel_error_expected metrics:
-    #
-    # Given 1 partition with probability_to_keep=0.4, actual_count=100,
-    # abs_error_expected=-50;
-    # -> abs_error_expected = (0.4*-50)/0.4=-50
-    # -> abs_error_expected_w_dropped_partitions = 0.4*-50+0.6*-100=-80
-    #
-    # When public partitions are used, these will be exactly equal to
-    # abs/rel_error_expected.
-    with_dropped_partitions: float
+    # The following error metrics include error from dropped partitions for
+    # private partition selection. For example rmse:
+    #    rmse = sqrt(E(actual-output)^2) = f(actual-output).
```

Comment:
Do we not want to use `actual_value` and `dp_value` here (and below)?

Reply:
yes, it's better, done